man/hammer.1.ronn

```diff
@@ -0,0 +1,64 @@
+Hammer - the CLI tool (not only) for Foreman
+============================================
+
+DESCRIPTION
+-----------
+
+Hammer is a generic [clamp-based](https://github.com/mdub/clamp) CLI framework.
+Hammer-cli provides just the core functionality. The core is extensible using plugins that contain application-specific commands.
+
+This architecture allows for easy customization according to your application. Nearly any Ruby script can be turned into a Hammer command, so the possibilities are endless. You also can easily add custom commands for your specific use, such as bulk actions or admin tasks.
```

Comment:
Copied from https://github.com/theforeman/hammer-cli/blob/master/README.md
If you feel this is irrelevant I'll remove it.